packages/ripple-keypairs/src/Sha512.ts

```diff
@@ -1,39 +1,36 @@
-/* eslint-disable no-bitwise --
- * lots of bitwise operators necessary for this */
-import * as hashjs from 'hash.js'
-import BigNum = require('bn.js')
+import { sha512 } from '@xrplf/isomorphic/sha512'
+import { bytesToNumberBE } from '@noble/curves/abstract/utils'
+
+type Input = Uint8Array | number[] | string
 
 export default class Sha512 {
-  // TODO: type of `hash`?
-  hash: any
+  // instantiate empty sha512 hash
+  hash = sha512.create()
```

Comment:
Why is this not done in a constructor anymore?

Reply:
Properties of a class are initiated in a constructor.